listenbrainz_spark/stats/user.py

```diff
@@ -131,42 +148,55 @@ def main(app_name):
     Args:
         app_name: Application name to uniquely identify the submitted
                   application amongst other applications
+
+        The script will be run to calculate stats for previous month.
+
     """
     t0 = time.time()
     try:
         listenbrainz_spark.init_spark_session(app_name)
     except Exception as err:
         logging.error("Cannot initialize spark session: %s / %s. Aborting." % (type(err).__name__, str(err)))
-        sys.exit(0)
+        sys.exit(-1)
     date = date_for_stats_calculation()
     try:
         df = listenbrainz_spark.sql_context.read.parquet('{}/data/listenbrainz/{}/{}.parquet'.format(config.HDFS_CLUSTER_URI, date.year, date.month))
         print("Loading dataframe...")
     except Exception as err:
         logging.error("Cannot read files from HDFS: %s / %s. Aborting." % (type(err).__name__, str(err)))
-        sys.exit(0)
+        sys.exit(-1)
     df.printSchema()
-    print(df.columns)
-    print(df.count())
+
     table = 'listens_{}'.format(datetime.strftime(date, '%Y_%m'))
     print(table)
     df.registerTempTable(table)
     print("Running Query...")
     query_t0 = time.time()
     print("DataFrame loaded in %.2f s" % (query_t0 - t0))
+
     data = defaultdict(dict)
+    """
```

Comment:
Comments and docstrings are different in python. 

https://stackoverflow.com/a/19076001

This should probably be a comment using hashes (`#`) and not docstrings.